### PR TITLE
Convert kops-aws jobs to bootstrap

### PR DESCRIPTION
**Pull request by krzyzacy**
ref #1026 #1058

hope to fix https://github.com/kubernetes/kubernetes/issues/36328

@fejta I did not take care of things like `disabled: '{obj:disable_job}'`, let me know if I need to create a bootstrap-ci-kops.yaml instead.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1209)

### CODE REVIEW

**Comment by nikhiljindal** [comment]
@fejta @k8s-oncall Can you please merge this? It claims to fix https://github.com/kubernetes/kubernetes/issues/36328 which is blocking the merge queue.

File: jobs/ci-kubernetes-e2e-kops-aws.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+export AWS_CONFIG_FILE="/workspace/.aws/credentials"
+# This is needed to be able to create PD from the e2e test
+export AWS_SHARED_CREDENTIALS_FILE="/workspace/.aws/credentials"
+# TODO(zmerlynn): Eliminate the other uses of this env variable
+export KUBE_SSH_USER=admin
+export LOG_DUMP_USE_KUBECTL=yes
+export LOG_DUMP_SSH_KEY=/workspace/.ssh/kube_aws_rsa
+export LOG_DUMP_SSH_USER=admin
+export LOG_DUMP_SAVE_LOGS=cloud-init-output
+{job-env}
```
Comment (fejta) [inline jobs/ci-kubernetes-e2e-kops-aws.sh]: Looks like bad copy-pasting here and elsewhere.

**Comment by fejta** [comment]
Merging it right now will not help things as we also need a PR to update the job the SQ follows. And looking it over more carefully to try and understand why this will make the job pass I noticed some other errors.

File: jobs/ci-kubernetes-e2e-kops-aws-updown.sh [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+export KOPS_PUBLISH_GREEN_PATH="gs://kops-ci/bin/latest-ci-updown-green.txt"
+
+### post-env
+
+# Assume we're upping, testing, and downing a cluster
+export E2E_UP="${E2E_UP:-true}"
+export E2E_TEST="${E2E_TEST:-true}"
+export E2E_DOWN="${E2E_DOWN:-true}"
+
+export E2E_NAME='bootstrap-e2e'
```
Comment (zmerlynn) [inline jobs/ci-kubernetes-e2e-kops-aws-updown.sh]: This can't be right?

File: jobs/ci-kubernetes-e2e-kops-aws.sh [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+export KOPS_PUBLISH_GREEN_PATH="gs://kops-ci/bin/latest-ci-green.txt"
+
+### post-env
+
+# Assume we're upping, testing, and downing a cluster
+export E2E_UP="${E2E_UP:-true}"
+export E2E_TEST="${E2E_TEST:-true}"
+export E2E_DOWN="${E2E_DOWN:-true}"
+
+export E2E_NAME='bootstrap-e2e'
```
Comment (zmerlynn) [inline jobs/ci-kubernetes-e2e-kops-aws.sh]: And here

File: jobs/ci-kubernetes-e2e-kops-aws.sh (hunk 1 above)
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-kops-aws.sh]: yes I misplaced it as provider env, nice catch

File: jobs/ci-kubernetes-e2e-kops-aws-updown.sh (hunk 2 above)
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-kops-aws-updown.sh]: ack it's defined above and should not be overwritten

**Comment by krzyzacy** [comment]
@nikhiljindal seems I was a liar.. :-(

**Comment by krzyzacy** [comment]
PTAL

**Comment by zmerlynn** [comment]
The Travis failure looks potentially real:
```
FAIL: testOnlyJobs (__main__.JobTest)
Ensure that everything in jobs/ is a valid job name and script.
```

**Comment by krzyzacy** [comment]
yup did not make it executable after re-do the whole file..

**Pull request merged**